Lib/test/test_genericalias.py

```diff
@@ -57,6 +57,10 @@
 from weakref import WeakSet, ReferenceType, ref
 import typing
 from typing import Unpack
+try:
+    from tkinter import Event # tkinter is not always available
+except ImportError:
+    Event = None
```

Comment:
This is not want we want. In case `tkinter` is not available, you test `None` to have the same features as `Event`. Instead, you should conditionally add `Event` to the list if it is not `None`, see:

```python
if ctypes is not None:
    generic_types.extend((ctypes.Array, ctypes.LibraryLoader))
```

Reply:
That's not necessary, `test_subscriptable()` checks for None already.

Reply:
Yeah, right now the code is checking for `None` when running the test and when the list with generic types is created. I think it's best if we just settle for one as the other check will be redundant at that point.

Reply:
It does, but still has no `None` values, if you delete these two lines and run the check - it will pass. But, it will fail with this change. Please, use a condition :)